vtex-apps/vertex
# (VERTEX-1) Add logs

**What problem is this solving?**
Need logs to debug tax amount discrepancies.

**How should this be manually tested?**
Use Vertex to calculate tax.  Confirm logging occurred.

```
curl --location --request POST 'brian--sandboxusdev.myvtex.com/vertex/checkout/order-tax' \
--header 'Content-Type: application/json' \
--data-raw '{
   "orderFormId":"7b32de86776246ba9677d446bae9f753",
   "salesChannel":"1",
   "items":[
      {
         "id":"0",
         "sku":"11",
         "ean":"2345234523453253",
         "refId":"1010109",
         "unitMultiplier":1,
         "measurementUnit":"un",
         "targetPrice":19.77,
         "itemPrice":19.77,
         "quantity":1,
         "discountPrice":-19.77,
         "dockId":"S0017-L0017-Dock",
         "freightPrice":0.0,
         "brandId":"2000001"
      },
      {
         "id":"1",
         "sku":"11",
         "ean":"2345234523453253",
         "refId":"1010109",
         "unitMultiplier":1,
         "measurementUnit":"un",
         "targetPrice":19.77,
         "itemPrice":19.77,
         "quantity":1,
         "discountPrice":0.0,
         "dockId":"S0017-L0017-Dock",
         "freightPrice":0.0,
         "brandId":"2000001"
      },
      {
         "id":"2",
         "sku":"2",
         "ean":"1233333",
         "refId":null,
         "unitMultiplier":1,
         "measurementUnit":"un",
         "targetPrice":40.0,
         "itemPrice":40.0,
         "quantity":1,
         "discountPrice":0.0,
         "dockId":"S0017-L0017-Dock",
         "freightPrice":0.0,
         "brandId":"2000001"
      }
   ],
   "totals":[
      {
         "id":"Items",
         "name":"Items Total",
         "value":7954.0
      },
      {
         "id":"Discounts",
         "name":"Discounts Total",
         "value":-1977.0
      },
      {
         "id":"Shipping",
         "name":"Shipping Total",
         "value":0.0
      },
      {
         "id":"Tax",
         "name":"Tax Total",
         "value":0.0
      }
   ],
   "clientEmail":null,
   "shippingDestination":{
      "country":"USA",
      "state":"NY",
      "city":"Williamson",
      "neighborhood":null,
      "postalCode":"14589",
      "street":"7533 Bear Swamp Rd"
   },
   "clientData":{
      "email":"brian.talma@vtex.com.br",
      "document":null,
      "corporateDocument":null,
      "stateInscription":null
   },
   "paymentData":{
      "payments":[
         {
            "paymentSystem":"4",
            "bin":"555555",
            "referenceValue":6455.0,
            "value":6455.0,
            "installments":1
         }
      ]
   }
}'
```
